daemon/container_windows.go

```diff
@@ -202,3 +200,9 @@ func (daemon *Daemon) mountVolumes(container *Container) error {
 func (container *Container) unmountVolumes(forceSyscall bool) error {
 	return nil
 }
+
+// reserveLabel reserves a label for the process label in the container. This
+// is a no-op on Windows
+func reserveLabel(container *Container) error {
```

Comment:
Hmm `label.ReserveLabel` already should be no-op for `!linux`:
https://github.com/opencontainers/runc/blob/master/libcontainer/label/label.go#L43